src/libpriv/rpmostree-origin.cxx

```diff
@@ -564,20 +586,7 @@ update_keyfile_pkgs_from_cache (RpmOstreeOrigin *origin,
       update_string_list_from_hash_table (origin->kf, group, key, pkgs);
     }
 
-  if (g_hash_table_size (pkgs) > 0)
-    {
-      switch (origin->refspec_type)
-        {
-        case RPMOSTREE_REFSPEC_TYPE_OSTREE:
-        case RPMOSTREE_REFSPEC_TYPE_CHECKSUM:
-          {
-            g_key_file_set_value (origin->kf, "origin", "baserefspec",
-                                  origin->cached_refspec);
-            g_key_file_remove_key (origin->kf, "origin", "refspec", NULL);
-            break;
-          }
-        }
-    }
```

Comment:
This chunk of code also handled overrides, but the new code only handles layered packages.